### ISSUE TITLE
Multi-line list item does not indent correctly

### ISSUE THREAD

**Issue by yungyuc** [sciwork/swportal#76]
See the blue text in t.he following picture:

![image](https://user-images.githubusercontent.com/399122/127728267-43a07c91-1448-49a9-a3e2-bbdc1c4314ff.png)

For some reason the style sheet does not have the multi-line list item to align well where it should indent.

**Comment by terrychan999** [comment]
The `<li>` is configured to `list-inside`
ref: https://github.com/sciwork/swportal/blob/92c77e58272baefd8d063129e630702e9f768fc6/tailwind/styles.css/#L37

these changes can solve the problem:
```css
ol,ul {
    padding-left: 1em;
}

li { @apply list-outside list-disc; }
```

demonstration:
![image](https://user-images.githubusercontent.com/14991138/128601215-ef14815d-4eed-408a-9f63-e143413ee4fc.png)

**Comment by yungyuc** [comment]
Thanks for the review!  If you have time, could you file a PR for a fix?

**Comment by yungyuc** [comment]
Fixed by #79
